### DISCUSSION TITLE
I can't see my apsystems gateway

### DISCUSSION THREAD

**Discussion by NoGrout** [ksheumaker/homeassistant-apsystems_ecur#112] [Q&A] [answered]
Hello, 
I installed HACS, restarted HA.
I added your repository, then restarted HA.
Added APSystems ECU-R package (with BETA), then restarted HA.

However, I don't see the entities in HACS, nor in HA.

Do the panels need to produce?
I can ping (lan IP) from my home assistant container my APSystems gateway with no problem.

Thanks for your help.

**Comment by HAEdwin** [comment, marked as answer]
It is preferably useful if the inverters are online when you install the integration.What happenend after applying the configuration steps? https://github.com/ksheumaker/homeassistant-apsystems_ecur#configuration

**Reply by NoGrout** [reply]
Hi, I'm bad, it's work with the sun ;)

I have an ECU-C, so I have to set up the reboot.
The automation command must be put in automations via HACS?
Or is it the same automations.yaml file from home assistant?
I hope I was clear enough. I'm new to HA.

The ID should not be changed? (in automation)
But the IP, yes if I understood correctly.
Thanks a lot

**Comment by HAEdwin** [comment]
Yes, use the IP-Address from your ECU-C. To put the automation in place you can either copy and paste the suggested automation from the help file into automations.yaml (using File Editor) but note the correct indent Or you could build the automation yourself using [Settings] > [Automations & Scenes] > [+ Create Automation], it will then still be added to automations.yaml. The ID can be left as-is if you do not build the automation yourself, it's just an automated unique ID.

**Comment by NoGrout** [comment]
I used the example code for automation but I actually have an indentation error.
Despite several modifications, I can't find a valid code. Can you see my mistake?
Thanks again!
![auto1](https://user-images.githubusercontent.com/96403302/193284475-49344ec3-2420-4bf2-af78-710593f3ebc5.PNG)
error: 
Error loading /config/configuration.yaml: expected '<document start>', but found '<block mapping start>'
in "/config/automations.yaml", line 8, column 1

**Reply by ViperRNMC** [reply]
line 8, remove the id line

**Reply by HAEdwin** [reply]
There were indent mistakes in the readme also. Corrected them just now.

**Reply by NoGrout** [reply]
Thank you ViperRNMC and HAEdwin.
Indeed, it works without ID. 
Probably because I don't currently have any other automation.

**Comment by NoGrout** [comment]
Last questions ^^
How to test the reboot?
If the information no longer reaches HA, does the photovoltaic system still work?
Thank you very much!

**Reply by NoGrout** [reply]
I found to do a test :D

**Reply by HAEdwin** [reply]
[Setting] > [Automations] and selecting the three dots on top of eachother then select [Run] 
Without the ECU the PV system should still do it's job although parameters exist to keep the installation within the prescribed requirements such as mains frequency and voltage.

**Reply by NoGrout** [reply]
Ok, thank you HAEdwin
